src/microbe_stage/MicrobeHUD.cs

```diff
@@ -432,10 +435,17 @@ protected override void UpdateAbilitiesHotBar()
             engulfing = control.State == MicrobeState.Engulf;
         }
 
+        bool isDigesting = false;
+
+        ref var engulfer = ref stage.Player.Get<Engulfer>();
+
+        if (engulfer.EngulfedObjects != null && engulfer.EngulfedObjects?.Count != 0)
```

Comment:
```suggestion
        if (engulfer.EngulfedObjects is {Count: > 0})
```

I think a single check like this is equivalent.